vue/components/forms/personalization-form/initials-images.vue

```diff
@@ -160,7 +160,7 @@ export const InitialsImages = {
          */
         openExternally: {
             type: Boolean,
-            default: false
+            default: null

```

Comment:
Why did you change this?

Reply:
I wanted to give priority to the prop, then the feature, then default to false.

If you default the prop to false, the feature would never be used.

Reply:
Got it!